celer/multitask_fast.pyx

```diff
@@ -70,7 +70,6 @@ cdef void set_prios_mtl(
         int * n_screened) nogil:
     cdef int j, k
     cdef int inc = 1
-    cdef floating tmp
```

Comment:
Forgive these changes. (Might be considered out of the scoop of the PR)
They shall be fixed here or never!